js/main.js

```diff
@@ -321,8 +347,80 @@ function createReview(event) {
   for (let i = 0; i < $stars.length; i++) {
     $stars[i].className = 'far fa-star star-icon';
   }
+  $reviewList.prepend(renderReview(reviewObj));
+  var $noReviewsMsg = document.querySelector('#no-reviews-msg');
+  $noReviewsMsg.className = 'column-full text-align-center hidden';
+  data.view = 'reviews-view';
+  $reviewsView.className = '';
+  $reviewFormView.className = 'hidden';
+  $searchView.className = 'hidden';
+  $resultsView.className = 'hidden';
   $starRating.setAttribute('data-star', '0');
   $reviewForm.reset();
   resetSearchBar();
   removeSearchResults();
 }
+
+function renderReview(reviewObj) {
+  var $liEl = document.createElement('li');
+  $liEl.className = 'column-half-review marg-1-1-2-bot text-align-center review-card';
+  $liEl.setAttribute('data-review-id', reviewObj.revieId);
+  var $divEl1 = document.createElement('div');
+  $divEl1.className = 'pad-1-bot';
+  var $imgEl = document.createElement('img');
+  $imgEl.className = 'review-img';
+  $imgEl.setAttribute('src', reviewObj.posterUrl);
+  var $divEl2 = document.createElement('div');
+  $divEl2.className = 'star-rating-review';
+  $divEl2.setAttribute('data-star', '0');
+
+  var $starAmt = parseInt(reviewObj.starRating);
+  for (let j = 0; j < $starAmt; j++) {
+    var $iEl1 = document.createElement('i');
+    $iEl1.className = 'fas fa-star star-icon-review';
+    $divEl2.appendChild($iEl1);
+  }
+  for (let j = $starAmt; j < 5; j++) {
+    var $iEl2 = document.createElement('i');
+    $iEl2.className = 'far fa-star star-icon-review';
+    $divEl2.appendChild($iEl2);
+  }
+
+  var $divEl3 = document.createElement('div');
+  $divEl3.className = 'pad-1';
+  var $h3El = document.createElement('h3');
+  $h3El.className = 'no-marg review-title';
+  $h3El.textContent = reviewObj.title;
+  var $pEl = document.createElement('p');
+  $pEl.className = 'review-notes pad-2-bot';
+  $pEl.textContent = reviewObj.reviewNotes;
+  // var $divEl4 = document.createElement('div');
+  // var $iEl6 = document.createElement('i');
+  // $iEl6.className = 'fas fa-pen-alt pen-icon';
+
+  $divEl1.appendChild($imgEl);
+  $divEl3.appendChild($h3El);
+  $divEl3.appendChild($pEl);
+  // $divEl4.appendChild($iEl6);
+  $liEl.appendChild($divEl1);
+  $liEl.appendChild($divEl2);
+  $liEl.appendChild($divEl3);
+  // $liEl.appendChild($divEl4);
```

Comment:
Please remove commented code prior to submitting for review

Reply:
Removed!